admin-dev/themes/new-theme/js/app/components/checkboxes-dropdown/CheckboxesDropdown.vue

```diff
@@ -72,66 +71,52 @@
 
 <script lang="ts">
   import {defineComponent, PropType} from 'vue';
-  import ProductEventMap from '@pages/product/product-event-map';
-
-  const CombinationEvents = ProductEventMap.combinations;
+  import {Choice} from '@app/components/checkboxes-dropdown/types';
 
   export default defineComponent({
-    name: 'FilterDropdown',
-    data(): {selectedFilters: Array<Record<string, any>>} {
-      return {
-        selectedFilters: [],
-      };
-    },
     props: {
       parentId: {
         type: Number,
-        required: true,
+        default: 1,
```

Comment:
If it's now an optional prop maybe it's default value should be falsy (0, null or false I don't know)

Reply:
There is still logic using elements based on parentId, so I didn't think its worth complicating things by changing this value.